src/components/Projects/Echo.tsx

```diff
@@ -0,0 +1,28 @@
+import { AccentText, BodyText } from 'components/Text'
+import EchoLogo from 'icons/EchoLogo'
+import ProjectBlock from 'components/Projects/ProjectBlock'
+
+export default function () {
+  return (
+    <ProjectBlock
+      logo={<EchoLogo />}
+      titleColor="text-secondary"
+      titleText="Echo"
+      subtitle="Tweet anonymously using your ZK badges"
+      description={
+        <BodyText>
+          <AccentText color="text-accent">SealCred</AccentText>{' '}
+          <AccentText color="text-secondary">| Echo</AccentText> allows you to
+          spill the tea using a pseudonymous identity built from your ZK badges.
+          Imagine creating a ZK badge of your work email and tweeting 100%
+          anonymously O_O
+        </BodyText>
```

Comment:
May we move this description into a separate component? I think it will be easier to read it

Reply:
Yup, let's put it near